tests/gordo_components/dataset/test_dataset.py

```diff
@@ -173,21 +178,25 @@ def test_row_filter(self):
         start = dateutil.parser.isoparse("2017-12-25 06:00:00Z")
         end = dateutil.parser.isoparse("2017-12-29 06:00:00Z")
         X, _ = TimeSeriesDataset(
-            MockDataSource(), start, end, tag_list=tag_list
+            data_provider=MockDataSource(), from_ts=start, to_ts=end, tag_list=tag_list
         ).get_data()
 
         self.assertEqual(577, len(X))
 
         X, _ = TimeSeriesDataset(
-            MockDataSource(), start, end, tag_list=tag_list, row_filter="'Tag 1' < 5000"
+            data_provider=MockDataSource(),
+            from_ts=start,
+            to_ts=end,
+            tag_list=tag_list,
+            row_filter="'Tag 1' < 5000",
         ).get_data()
 
         self.assertEqual(8, len(X))
 
         X, _ = TimeSeriesDataset(
-            MockDataSource(),
-            start,
-            end,
+            data_provider=MockDataSource(),
+            from_ts=start,
+            to_ts=end,
             tag_list=tag_list,
             row_filter="'Tag 1' / 'Tag 3' < 0.999",
         ).get_data()
```

Comment:
Not saying it should be done in this PR, but we should might think about using pytest to parameterize this instead, it looks like unnecessary duplication.